tests/test_connector_webexteams.py

```diff
@@ -0,0 +1,161 @@
+        }
+        message = amock.Mock()
+        connector.api.messages.get = amock.Mock()
+        message.text = "Hello"
+        message.roomId = "90ABCrWgrzfcz7LZi"
+        message.roomType = "general"
+        connector.api.messages.get.return_value = message
+        connector.get_person = amock.CoroutineMock()
+        person = amock.CoroutineMock()
+        person.displayName = "Himanshu"
+        connector.get_person.return_value = person
+
+        response = await connector.webexteams_message_handler(request)
+        self.assertLogs("_LOGGER", "debug")
+        self.assertEqual(201, response.status)
+        self.assertEqual('"Received"', response.text)
+        self.assertTrue(connector.api.messages.get.called)
+        self.assertTrue(connector.get_person.called)
+
+        connector.opsdroid = amock.CoroutineMock()
+        connector.opsdroid.parse = amock.CoroutineMock()
+        connector.opsdroid.parse.side_effect = KeyError
+        await connector.webexteams_message_handler(request)
+        self.assertLogs("_LOGGER", "error")
+
+    async def test_connect_fail_keyerror(self):
+        connector = ConnectorWebexTeams({})
+        connector.clean_up_webhooks = amock.CoroutineMock()
+        connector.subscribe_to_rooms = amock.CoroutineMock()
+        connector.set_own_id = amock.CoroutineMock()
+        await connector.connect(opsdroid=OpsDroid())
+        self.assertLogs("_LOGGER", "error")
+
+    async def test_listen(self):
+        """Test the listen method.
+
+        The Webex Teams connector listens using an API endoint and so the listen
+        method should just pass and do nothing. We just need to test that it
+        does not block.
+
+        """
+        connector = ConnectorWebexTeams({})
+        self.assertEqual(await connector.listen(opsdroid=OpsDroid()), None)
+
+    async def test_respond(self):
+        connector = ConnectorWebexTeams({"access-token": "abc123"})
+        connector.api = amock.CoroutineMock()
+        connector.api.messages.create = amock.CoroutineMock()
+        message = amock.CoroutineMock()
+        message.room.return_value = {"id": "3vABZrQgDzfcz7LZi"}
+        message.text.return_value = "Hello"
+        await connector.respond(message)
+        self.assertTrue(connector.api.messages.create.called)
+
+    async def test_get_person(self):
+        connector = ConnectorWebexTeams({"access-token": "abc123"})
+        connector.api = amock.CoroutineMock()
+        connector.api.messages.create = amock.CoroutineMock()
+        connector.api.people.get = amock.CoroutineMock()
+        connector.api.people.get.return_value = "Himanshu"
+        self.assertEqual(len(connector.people), 0)
+        await connector.get_person("3vABZrQgDzfcz7LZi")
+        self.assertEqual(len(connector.people), 1)
+
+    async def test_subscribe_to_rooms(self):
+        connector = ConnectorWebexTeams(
+            {"access-token": "abc123", "webhook-url": "http:\\127.0.0.1"}
```

Comment:
I dont think this url is valid

```suggestion
            {"access-token": "abc123", "webhook-url": "http://127.0.0.1"}
```